pkg/apic/client.go

```diff
@@ -87,15 +87,19 @@ type Client interface {
 	GetAccessControlList(aclName string) (*mv1a.AccessControlList, error)
 	UpdateAccessControlList(acl *mv1a.AccessControlList) (*mv1a.AccessControlList, error)
 	CreateAccessControlList(acl *mv1a.AccessControlList) (*mv1a.AccessControlList, error)
-	UpdateResourceInstance(ri *v1.ResourceInstance) (*v1.ResourceInstance, error)
-	DeleteResourceInstance(ri *v1.ResourceInstance) error
+
 	CreateSubResource(rm v1.ResourceMeta, subs map[string]interface{}) error
 	GetResource(url string) (*v1.ResourceInstance, error)
-	CreateResource(url string, bts []byte) (*v1.ResourceInstance, error)
-	UpdateResource(url string, bts []byte) (*v1.ResourceInstance, error)
 	UpdateResourceFinalizer(ri *v1.ResourceInstance, finalizer, description string, addAction bool) (*v1.ResourceInstance, error)
-	CreateOrUpdateResource(v1.Interface) (*v1.ResourceInstance, error)
 	IsMarketplaceSubsEnabled() bool
+
+	UpdateResourceInstance(ri v1.Interface) (*v1.ResourceInstance, error)
```

Comment:
Updated ResourceInstance methods on the client to take in a `v1.Interface`, and deprecated the `CreateResource` and `UpdateResource`